### PR TITLE
Bump regex dependency to 2021.4.4 to fix import of Pattern class

### PR DESCRIPTION
**Pull request by rowanseymour**
Fixes https://github.com/psf/black/issues/2620

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]
Thanks!

**Comment by JelleZijlstra** [comment]
Could you add a changelog entry? I'm leaning towards making a new release for this, so we should announce the change.

**Comment by JelleZijlstra** [comment]
People on Discord brought up that we also need to update Pipfile.lock.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 21.11b1
```
Comment (JelleZijlstra) [inline CHANGES.md:3]:
```suggestion
## Unreleased
```
We'll change this when we actually release

**Comment by rowanseymour** [comment]
@JelleZijlstra took a few goes to figure out pipenv locking but I think I have it right now..

**Review by ichard26 — APPROVED** [review]
Pipenv is a pain but this looks good enough? Thank you for noticing the original issue by the way @rowanseymour!

**Comment by rowanseymour** [comment]
thanks for freeing us from worrying about how format python!

**Pull request merged**